src/urdf_filter.cpp

```diff
@@ -278,9 +275,10 @@ void RealtimeURDFFilter::filter_callback
   // publish processed depth image and image mask
   if (depth_pub_.getNumSubscribers() > 0)
   {
-    cv::Mat masked_depth_image (height_, width_, CV_32FC1, masked_depth_);
+    cv::Mat masked_depth_image = masked_depth_;
```

Comment:
This is assigned twice.

Reply:
The data is cloned if the encoding has to be modified and passed by reference otherwise.